<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>272</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel {
	font-family: Poppins;
	font-size: 16px;
}

QWidget {
	background-color: white;
}

QPushButton {
	border: none;
	background-color: white;
	font-size: 14px;
	font-family: Poppins;
	font-weight: 500;
}

</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="backBtn">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>230</y>
      <width>240</width>
      <height>42</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: white;
	color: #266676;
	font-size: 16px;
	border: 1px solid #266676;
}</string>
    </property>
    <property name="text">
     <string>Atrás</string>
    </property>
   </widget>
   <widget class="QPushButton" name="nextBtn">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>230</y>
      <width>240</width>
      <height>42</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #1a759f;
	color: white;
	font-size: 16px;
	font-weight: 500;
	height: 50px;
	border: none;
}</string>
    </property>
    <property name="text">
     <string>Siguiente</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-1</y>
      <width>481</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Seleccione una opción</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QListView" name="originList">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>401</width>
      <height>161</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: none;</string>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
   </widget>
   <widget class="QSlider" name="verticalSlider">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>60</y>
      <width>41</width>
      <height>161</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QSlider::groove:vertical{ 
	background-color: rgb(234, 234, 234);
	height: 161px;
	border-radius: 10px;
}

QSlider::handle:vertical { 
	background-color: #22577a;
    height: 50;
    width: 20px;
    line-height: 10px; 
	margin-top: 0px; 
	margin-bottom: 0px;
	border-radius: 10px; 
}

QSlider{
	background-color: rgb(234, 234, 234);
	border-radius: 20px;
}

</string>
    </property>
    <property name="pageStep">
     <number>1</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <property name="invertedAppearance">
     <bool>true</bool>
    </property>
    <property name="invertedControls">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
